portal-impl/src/com/liferay/portal/action/UpdateLicenseAction.java

```diff
@@ -41,47 +44,46 @@ public ActionForward execute(
 			HttpServletResponse httpServletResponse)
 		throws Exception {
 
-		// PLACEHOLDER 01
```

Comment:
The placeholders were introduced by [LPSA-72555](https://liferay.atlassian.net/browse/LPSA-72555) so I think they might be still required.

Reply:
Very interesting 👍🏻 

I'll bring it back